eahub/templates/base.html

```diff
@@ -137,6 +137,25 @@ <h4> /{{ page_name }}</h4>
     {% endblock %}
 
     <main>
+
+      <br>
+      {% if messages %}
+      <div class="container mb-5 mt-5">
+        <div class="row">
+          <!-- remove the col-xs-10 row if want full width -->
+          <div class="col-m-4"> 
```

Comment:
col-m-* - doesn't seem like a valid size name?

Reply:
I guess we might need to use the bs5 version? that looks like the code from bs3?